sympy/stats/tests/test_rv.py

```diff
@@ -152,6 +152,27 @@ def test_given():
     assert X == A == B
 
 
+def test_factorial_moment():
+    X = Poisson('x', 2)
```

Comment:
This could be `X = Poisson('X', 2)` so that it is distinguishable from `Symbol('x')` being used for the symbolic tests. (Similarly for `Y = Binomial('Y', 2, 0.5)` or `Y = Binomial('Y', 2, S.Half)`)